src/main/scala/no/ndla/articleapi/service/converters/SimpleTagConverter.scala

```diff
@@ -33,23 +33,16 @@ object SimpleTagConverter extends ConverterModule {
         case "paragraph" => replaceTag(el, "section")
         case "quote" => replaceTag(el, "blockquote")
         case "hide" => handle_hide(el)
+        case "frame" => {
+          el.removeClass("frame")
+          el.addClass("c-bodybox")
```

Comment:
For at denne skal bli med må det også legges til en entry i [attributt-whitelisten](https://github.com/NDLANO/article-api/blob/master/src/main/resources/html-rules.json#L2). Da vil alle class-attributter bli tillatt for `div`s og ikke strippet vekk by default.
Vi vil fremdeles strippe bort alt annet av div-klasser, så da må det legges til en `el.removeAttribute("class")` under default casen på linje 41